apps/evm/lib/evm/configuration/frontier_test.ex

```diff
@@ -0,0 +1,15 @@
+defmodule EVM.Configuration.FrontierTest do
+  defstruct contract_creation_cost: 21_000, has_delegate_call: false
+
+  def new do
+    %__MODULE__{}
+  end
+end
+
+defimpl EVM.Configuration, for: EVM.Configuration.FrontierTest do
+  @spec contract_creation_cost(EVM.Configuration.t()) :: integer()
+  def contract_creation_cost(config), do: config.contract_creation_cost
+
+  @spec has_delegate_call?(EVM.Configuration.t()) :: boolean()
+  def has_delegate_call?(config), do: config.has_delegate_call
```

Comment:
No need to change this if you don't want to, but we could just change this to, 

```elixir
def has_delegate_call?(_config), do: false
```

There's no real reason why it needs to live in a `struct`, right?

Reply:
@germsvel yes. good idea